hal/armv7m/imxrt/10xx/hal.c

```diff
@@ -96,8 +97,23 @@ addr_t hal_kernelGetAddress(addr_t addr)
 }
 
 
+int hal_memAddMap(addr_t start, addr_t end, u32 attr, u32 id)
+{
+	return mpu_mapAlloc(start, end, attr, id);
+}
+
+
+void hal_memInfo(void)
+{
+	mpu_mapInfo();
+}
+
+
```

Comment:
This function is redundant. All information about maps should be displayed by map command belonging to hardware independent part.

Reply:
Which command should display hardware dependent setup (e.g. region setup?) like on the screenshot attached in PR introduction above.

Reply:
none

Reply:
Ok, I'll remove that.

Reply:
Thx. This is very important to strictly separate functionality of hardware dependent and independent layers.

Reply:
Removed.